lib/ansible/module_utils/network/eos/eos.py

```diff
@@ -409,7 +409,10 @@ def is_eapi(module):
 
 
 def to_command(module, commands):
-    if is_eapi(module):
+    if isinstance(commands, dict):
```

Comment:
`transform(to_list(commands))` already does this, with the bonus of not breaking if `output` is not specified in the dictionary. You should not have to change this file for this to work.

Reply:
Sorry, my bad I'll revert the change and will upload again